spec/lib/hesa/institution_spec.rb

```diff
@@ -5,37 +5,48 @@
     it 'returns a list of HESA institution structs' do
       institutions = described_class.all
 
-      expect(institutions.size).to eq 345
```

Comment:
As I understand it (from Ed Horsford) the reduced number of institutions in the new list is because the old one contained outdated ones that had merged / closed; a check against the existing data in prod was done to see if any existing data used them, and they didn't, so it was deemed safe to reduce the list. However, we can certainly add "dead" institutions and mark them as closed if they're useful for something and wouldn't just clutter the list!

Reply:
Since a check against data in all cycles in prod was done and we have covered all the institutions in this list, there's def no need to add extra ones, thanks